### PR TITLE
Scripts cleanup and documentation update

### PR DESCRIPTION
**Pull request by Michkov**
- Removed unused scripts
- Removed Development mode since preview mode is being developed
  actively
- Removed old scripts for build which were related to Milestone 2
- Updated README files
  - Separated gitops README file
  - Updated build README file and removed duplicities from main README
    file
  - Added info about setting tekton bundles
  - Added info about shared secrets

For easy to see you can check directly my branch with the change -> https://github.com/Michkov/infra-deployments/tree/cleanup

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,59 +54,62 @@ Simply update the files under `components/(team-name)`, and open a PR with the c
...
-
 SPI Vault instance has to be manually initialized. There is a script to help with that:
+
 1) Make sure that your cluster user has at least permissions `./components/spi/vault_role.yaml`
 2) Clone SPI operator repo `git clone https://github.com/redhat-appstudio/service-provider-integration-operator && cd service-provider-integration-operator`
 3) run `vault-init.sh` script from repo root directory `./hack/vault-init.sh`
 
-### Use SharedSecret with Tekton Chains
-
-During the build pipeline, it is possible to use the `redhat-appstudio-user-workload`
```
Comment (lcarva) [inline README.md:93]: Accidental removal? I thought this would be helpful to others.

Comment (lcarva) [reply README.md:93]: Oh you moved it to the right place. Thanks!

**Review by lcarva — APPROVED** [review]
LGTM! Someone else should also review this :)

File: components/build/README.md [hunk 2]
```diff
@@ -1,80 +1,93 @@
...
 
-```
-./hack/build/ls-all-my-repos.sh | xargs -n 1 ./hack/build/check-repo.sh
-```
\ No newline at end of file
+| Name | Source | Description | Access |
+| -- | -- | -- | -- |
+| test-team-snyk | test-teams-snyk secret in test-team namespace | Snyk token used by HACBS pipelines | users/serviceaccounts with edit role |
+| redhat-appstudio-user-workload | redhat-appstudio-user-workload secret in build-templates namespace | Quay secret allowing to push into default AppStudio repository | users/serviceaccounts with edit role |
+| redhat-appstudio-staginguser | redhat-appstudio-staginguser secret in build-templates namespace | Quay secret allowing to push into component repositories in redhat-appstudio org | `pipeline` service accounts defined in [shared-resources-components.yaml](https://github.com/redhat-appstudio/infra-deployments/blob/main/components/build/shared-resources/shared-resources-components.yaml)|
```
Comment (gabemontero) [inline components/build/README.md:83]: @Michkov I think the existing doc had a gap wrt `redhat-appstudio-registry-pull-secret` (I had pinged @sbose78 about that in his original doc update PR but it never got resolved).

In particular, both the the use of the shared secret as well as the bypassing the use of the shared secret if you define `redhat-appstudio-registry-pull-secret` in your namespace.

All part of PLNSRVCE-250, with https://github.com/redhat-appstudio/application-service/pull/116 and https://github.com/redhat-appstudio/build-service/pull/29

Can we add those details to the doc as part of your cleanup?

thanks

**Review by zregvart — APPROVED** [review]
:+1: Thank you!

File: components/build/README.md (hunk 2 above)
Comment (Michkov) [reply components/build/README.md:83]: @gabemontero added https://github.com/Michkov/infra-deployments/blob/cleanup/components/build/README.md#repository-secrets

Comment (gabemontero) [reply components/build/README.md:83]: looks good thanks @Michkov

**Review by gabemontero — APPROVED** [review]

File: hack/preview-template.env [hunk 3]
```diff
@@ -4,13 +4,15 @@
...
-
 ## HAS enable github integration
 ### Your GITHUB organization where to manage repositories by HAS
 export MY_GITHUB_ORG=
 ### Personal API token with repo and delete_repo permission
 export MY_GITHUB_TOKEN=
+
+# Optional
+
+## HAS enable github integration
```
Comment (brunoapimentel) [inline hack/preview-template.env:15]: Was this line copied accidentally?

Comment (Michkov) [reply hack/preview-template.env:15]: nope, it's optional part for `HAS enable github integration`, so it has both optional and required

**Pull request merged**